src/libstore/content-address.hh

```diff
@@ -55,10 +86,10 @@ std::optional<ContentAddress> parseContentAddressOpt(std::string_view rawCaOpt);
 
 Hash getContentAddressHash(const ContentAddress & ca);
 
-/*
-  We only have one way to hash text with references, so this is single-value
-  type is only useful in std::variant.
-*/
+/**
+ * We only have one way to hash text with references, so this is
+ * single-value type is only useful in std::variant.
+ */
```

Comment:
```suggestion
/*
 * We only have one way to hash text with references, so this is
 * single-value type is only useful in std::variant.
 */
```

This comment might be useful as a code comment, but I don't think it makes much sense in the API documentation